data/us_census_2000_male_first.txt

```diff
@@ -1,4 +1,4 @@
-JAMES          3.318  3.318      1
+AARON          3.318  3.318      1
```

Comment:
Are you for realz?

On 24 April 2015 at 16:42, Philip Jackson <notifications@github.com> wrote:

> In data/us_census_2000_male_first.txt
> <https://github.com/adi-pjackson/zxcvbn/pull/1#discussion_r29023424>:
>
> > @@ -1,4 +1,4 @@
> > -JAMES          3.318  3.318      1
> > +AARON          3.318  3.318      1
>
> Should be Aron
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adi-pjackson/zxcvbn/pull/1/files#r29023424>.
>